install_config/install/deploy_router.adoc

```diff
@@ -635,6 +638,23 @@ $ oadm router \
 ====
 endif::[]
 
+
+=== F5 Router Partition Paths
+Partition paths allow you to store your OpenShift routing configuration in a
+custom *F5 BIG-IP®* administrative partition instead of the
+default */Common* partition.  Custom administrative partitions can be used
+to secure *F5 BIG-IP®* environments so that all OpenShift specific
+configuration stored in *F5 BIG-IP®* system objects resides within a
```

Comment:
Ok, then the problem might be with "all". If there's only one configuration, then maybe "...so that the OpenShift-specific configuration stored in...". If I'm understanding this correctly the sentence goes from plural (all) to singular (configuration).